resource_estimator/src/system/tests.rs

```diff
@@ -472,21 +472,21 @@ pub fn test_hubbard_e2e_increasing_max_num_qubits() -> Result<()> {
 fn prepare_chemistry_estimation_with_expected_majorana(
 ) -> PhysicalResourceEstimation<Protocol, TFactoryBuilder, LogicalResourceCounts> {
     let ftp = Protocol::floquet_code();
-    let qubit = Rc::new(PhysicalQubit::qubit_maj_ns_e4());
+    let qubit = Rc::new(PhysicalQubit::qubit_maj_ns_e6());
```

Comment:
Out of curiosity, why change this to the more precise Majorana qubit? Likewise the error budget below is more tolerant. Is there value in keeping both test cases or is the updated the one the only one worth testing?

Reply:
I wanted to add a scenario in which the error with the factory calculation is observed. The other related test cases that also use a chemistry benchmark just need _some_ benchmark, so it might as well be this one. I don't think there is value in having both. Therefore, for brevity, I decided to use this one for the other test cases as well.